tmk_core/common/eeconfig.c

```diff
@@ -48,6 +48,13 @@ void eeconfig_init_quantum(void) {
     eeprom_update_dword(EECONFIG_RGB_MATRIX, 0);
     eeprom_update_byte(EECONFIG_RGB_MATRIX_SPEED, 0);
 
+    // TODO: Remove once ARM has a way to configure EECONFIG_HANDEDNESS
+    //        within the emulated eeprom via dfu-util or another tool
+#ifdef INIT_EE_HANDS_LEFT
```

Comment:
What happens if this was set incorrectly, and then reflashed? 

Should probably set it to 0, for ARM split boards.. 

eg: 
```c
#ifdef __arm__
#    ifdef INIT_EE_HANDS_LEFT
#    pragma message "Faking EE_HANDS for left hand"
    eeprom_update_byte(EECONFIG_HANDEDNESS, 1);
#    else
#    pragma message "Faking EE_HANDS for right hand"
    eeprom_update_byte(EECONFIG_HANDEDNESS, 0);
#    endif
#endif
```

Reply:
Tried to not lock down the logic too much so it *could* be used for non dfu-util non arm if required. For constancy I agree its best to init the right hand till eeprom memory is zeroed on all platforms.